data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1625 @@
+			`	Before long the technicians and pilots are trading stories while the Mergansers overshadow the group. Sometimes they poke various engineers or technicians, and the technicians frequently share messages from their commlinks. Apparently they have them hooked up to the Mergansers, so the ships are sharing their own accounts of the patrol. After several hours of the festivities, Taely announces that everyone needs to get to sleep because tomorrow is a new day. As everyone bids their farewells, you note that they include the ships, with most people patting or stroking one or more of the ships on their way out.`
+			`	"Stop by the cafeteria tomorrow morning, <first>. There will be more you can help with, if you are interested." With that, she gestures farewell and heads off at a brisk pace.`
+
+
+
+mission "Remnant: Cognizance 24"
+	name "Equipment to Ssil Vida"
+	description "Dusk needs a load of <cargo> taken to Ssil Vida."
+	cargo "Outpost Equipment" 11
+	passengers 1
+	source
+		government "Remnant"
+	stopover "Esquiline"
+	destination "Ssil Vida"
+	to offer
+		has "Remnant: Cognizance 23: done"
+	on offer
+		conversation
+			`When you enter the cafeteria you are greeted by Dusk and Taely signing at you to join them from across the room. Collecting a tray from the dispenser, you slide into an empty seat at their table, watching as Taely signs: "We were just exchanging news and discovered we can combine our next assignment."`
+			`	Dusk continues. "Based on the initial data we collected while we were at Ssil Vida, there is sufficient novel data to deserve a permanent research team there. Conveniently, the station is already habitable, but we will need several shipments of equipment to setup a self-contained section."`
+			choice
+				`	"OK, clear enough."`
+					goto skipexplanation
+				`	"Why do you need to do that?"`
+			`	"For safety. The station might appear to be enduring and constant, but if we lacked independent facilities, the slightest failure could kill everyone. Given that we do not even understand how the station is running, depending on it moving forward would be foolish." Taely nods in agreement with Dusk, and adds. "We will, of course, be looking to do a full evaluation of the station eventually, but it is rather massive and it will be quite some time before we can finish that.`
+			label skipexplanation
+			`	Dusk gestures and their commlink sprays a holographic list into the air between you. "Basically, we would like you to load this set of equipment, then stop by Esquiline to pick up a technician by the name of Gavriil, who will be leading the establishment of the life support systems on Ssil Vida."`
```

Comment:
I was intentionally portraying Dusk as being somewhat Androgynous and/or ambiguous with referring to them as they/them. Does this work for doing that?

Reply:
Cool with me! The reason I suggested this change is because I remember Dusk already being referred to as a he/him (maybe in #6153). I'll double check/correct #6153 myself so Becca doesn't have to worry about it if you're cool with double checking this file for it.